.github/workflows/tiobe-scan.yaml

```diff
@@ -24,6 +24,23 @@ jobs:
         go install github.com/axw/gocov/gocov@v1.1.0
         go install github.com/AlekSi/gocov-xml@v1.1.0
 
+    - name: Install Docker
+      run: |
+        sudo apt-get update -y
+        sudo apt-get install -y \
+          ca-certificates \
+          curl \
+          gnupg \
+          lsb-release
+        sudo mkdir -p /etc/apt/keyrings
+        curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+        echo \
+          "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+          $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+        sudo apt-get update -y
+        sudo snap install docker # installing via apt doesn't work, because you can't start systemctl services.
```

Comment:
That's weird, why can't we start systemctl services? Let's also add a comment on the top of this step explaining why we're installing docker + compose

Reply:
Depending on the runner you use, you won't need to install docker. Are we using self-hosted here?